core/tests/webdriverio_desktop/extensions.js

```diff
@@ -76,7 +76,7 @@ describe('rich-text components', function() {
         await richTextChecker.readRteComponent(
           'Link', 'https://google.com/', true);
         await richTextChecker.readRteComponent(
-          'Video', 'M7lc1UVf-VE', 10, 100, false);
+          'Video', 'M7lc1UVf-VE', 10, 100, true);
```

Comment:
This changes the e2e test to expect an auto-playing video, but we don't change the video creation code. Why does this work?

Reply:
I somehow missed this comment, sorry about that. Yes you are correct, I have updated this part.
Thanks